Add missing quote (#23277)
dotnet/AspNetCore.Docs: aspnetcore/blazor/javascript-interoperability/call-javascript-from-dotnet.md

> [!NOTE]
> The following examples aren't typical use cases for this scenario because the [struct](/dotnet/csharp/language-reference/builtin-types/struct) passed to JS doesn't result in poor component performance. The example uses a small object merely to demonstrate the concepts for passing unserialized .NET data.

> Place the following `<script>` block in `wwwroot/index.html` (Blazor WebAssembly) or `Pages/_Layout.cshtml` (Blazor Server). Alternatively, you can place the JS in an external JS file referenced inside the closing `</body>` tag with `<script src="{SCRIPT PATH AND FILE NAME (.js)}"></script>`, where the `{SCRIPT PATH AND FILE NAME (.js)}` placeholder is the path and file name of the script.

Reviewer: ```suggestion
Place the following `<script>` block in `wwwroot/index.html` (Blazor WebAssembly) or `Pages/_Layout.cshtml` (Blazor Server). Alternatively, the JavaScript can be added to external file referenced inside the closing `</body>` tag. For example, `<script src="{SCRIPT PATH AND FILE NAME (.js)}"></script>`, where the `{SCRIPT PATH AND FILE NAME (.js)}` placeholder is the path and file name of the script.
```

Reviewer 2: I'm trying to avoid crazy too many state verbs. I'm actually not even sure where the whole 'avoid 2nd person' thing came from ...

https://docs.microsoft.com/en-us/style-guide/grammar/person#in-general-use-second-person

```suggestion
Place the following `<script>` block in `wwwroot/index.html` (Blazor WebAssembly) or `Pages/_Layout.cshtml` (Blazor Server). Alternatively, add the JS to an external file referenced inside the closing `</body>` tag. For example, `<script src="{SCRIPT PATH AND FILE NAME (.js)}"></script>`, where the `{SCRIPT PATH AND FILE NAME (.js)}` placeholder is the path and file name of the script.
```

Reviewer: You is dead wood. It's shorter without it. When Fowlers 3rd cousin, a php architect is reading our docs, she's not writing the code, she's consulting. So you doesn't apply to anyone who's reading it and not doing exactly that. I got it from Roth with ASP.NET Core 1.0, he wanted a more reference style.

Reviewer 2: Fair enough. I don't do it much, but I'm confused why they advise to use it so clearly. It feels like they must have had *many* meetings on it. 😄 Anyway ... imperative tone should be good. I use it a lot with success.
